tests/Fixer/LanguageConstruct/SingleSpaceAfterConstructFixerTest.php

```diff
@@ -0,0 +1,746 @@
+            'constructs' => [
+                'print',
+            ],
+        ]);
+
+        $this->doTest($expected, $input);
+    }
+
+    public function provideFixWithPrintCases()
+    {
+        return [
+            [
+                '<?php print 9000;',
+                '<?php print  9000;',
+            ],
+            [
+                '<?php print 9000;',
+                '<?php print
+
+9000;',
+            ],
+            [
+                '<?php print /* foo */9000;',
+                '<?php print/* foo */9000;',
+            ],
+        ];
+    }
+
+    /**
+     * @dataProvider provideFixWithRequireCases
+     *
+     * @param string      $expected
+     * @param null|string $input
+     */
+    public function testFixWithRequire($expected, $input = null)
+    {
+        $this->fixer->configure([
+            'constructs' => [
+                'require',
+            ],
+        ]);
+
+        $this->doTest($expected, $input);
+    }
+
+    public function provideFixWithRequireCases()
+    {
+        return [
+            [
+                '<?php require "vendor/autoload.php";',
+                '<?php require  "vendor/autoload.php";',
+            ],
+            [
+                '<?php require "vendor/autoload.php";',
+                '<?php require
+
+"vendor/autoload.php";',
+            ],
+            [
+                '<?php require /* foo */"vendor/autoload.php";',
+                '<?php require/* foo */"vendor/autoload.php";',
+            ],
+        ];
+    }
+
+    /**
+     * @dataProvider provideFixWithRequireOnceCases
+     *
+     * @param string      $expected
+     * @param null|string $input
+     */
+    public function testFixWithRequireOnce($expected, $input = null)
+    {
+        $this->fixer->configure([
+            'constructs' => [
+                'require_once',
+            ],
+        ]);
+
+        $this->doTest($expected, $input);
+    }
+
+    public function provideFixWithRequireOnceCases()
+    {
+        return [
+            [
+                '<?php require_once "vendor/autoload.php";',
+                '<?php require_once  "vendor/autoload.php";',
+            ],
+            [
+                '<?php require_once "vendor/autoload.php";',
+                '<?php require_once
+
+"vendor/autoload.php";',
+            ],
+            [
+                '<?php require_once /* foo */"vendor/autoload.php";',
+                '<?php require_once/* foo */"vendor/autoload.php";',
+            ],
+        ];
+    }
+
+    /**
+     * @dataProvider provideFixWithReturnCases
+     *
+     * @param string      $expected
+     * @param null|string $input
+     */
+    public function testFixWithReturn($expected, $input = null)
+    {
+        $this->fixer->configure([
+            'constructs' => [
+                'return',
+            ],
+        ]);
+
+        $this->doTest($expected, $input);
+    }
+
+    public function provideFixWithReturnCases()
+    {
+        return [
```

Comment:
Please add a test case with a multiline return value, e.g.:
```php
return
    $a
    && $b
;
```
IMO in this case the newline after `return` is legit and should not be removed. This probably applies to more keywords (e.g. `yield`).